Automattic/jetpack
# VideoPress: tweak and rename poster panel title

On this PR
* Keep the poster panel closed by default
* [Rename the panel title to `Poster and preview`.](https://github.com/Automattic/jetpack/issues/29278#issuecomment-1481759533)
* Fix a var name typo.

Fixes #

## Proposed changes:

* VideoPress: tweak and rename poster panel title

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:







* When beta extensions are enabled:
  * Confirm the panel is closed by default
  * Confirm  the panel title is `Poster and preview`

before and beta disabled | after and beta enabled
------|------
<img width="445" alt="Screen Shot 2023-03-24 at 20 13 42" src="https://user-images.githubusercontent.com/77539/227659804-072e59e8-323f-4ddd-866d-b4967b2b5840.png"> | <img width="631" alt="Screen Shot 2023-03-24 at 20 13 03" src="https://user-images.githubusercontent.com/77539/227659809-97d44ef6-80dc-4fa5-a7f7-82df0730ff18.png">
